MapboxNavigation/NavigationViewController.swift

```diff
@@ -147,10 +147,11 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
      */
     public var shouldManageApplicationIdleTimer = true
     
-    /**
-     Coordinate which is used to determine what associated building extrusion should be highlighted.
-     */
-    public var buildingExtrusionCoordinate: CLLocationCoordinate2D? = nil
+    public var highlightDestinationBuildings: Bool = false {
```

Comment:
I think exposing `highlightDestinationBuildings` is a good idea. I wasn't actually aware that `Waypoint` contains `targetCoordinate` which allows to store highlighted buildings coordinates.

Reply:
In that case, we should document that the developer should always pass in a `targetCoordinate`, even if it’s redundant with the `coordinate`, to get highlighted buildings. Hopefully that doesn’t have any downsides.

Reply:
Passing in the `coordinate` will most likely result in no building highlight. `coordinate` can be automatically written in if no `targetCoordinate` is supplied.